tests/installation/welcome.pm

```diff
@@ -105,6 +106,7 @@ sub run {
     # Add tag to check for https://progress.opensuse.org/issues/30823 "test is
     # stuck in linuxrc asking if dhcp should be used"
     push @welcome_tags, 'linuxrc-dhcp-question';
+    push @welcome_tags, 'import-known-untrusted-gpg-key';
```

Comment:
why this line and the other one below are not conditional? will this always handle untrusted gpg keys... is something that we want to do always?

Reply:
I would suggest to enable it for single scenario only, because now we won't notice if one of the normal repos gets broken signature.

Reply:
I believe that this is not gonna hide another unstrusted key. first you need the repo, and second the repo must have signature with 396020B4CD8F20FE. Although it makes sense to make it conditional.

Reply:
if get_var('INSTALLER_SELF_UPDATE') is added